catalog/CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ where verb is one of
 
 ## Changes
 
+- [Added] Select files and directories in packages to download ([#4173](https://github.com/quiltdata/quilt/pull/4173))
```

Comment:
```suggestion
- [Added] Selective package downloading ([#4173](https://github.com/quiltdata/quilt/pull/4173))
```
maybe sth like this?

Reply:
I started with this, but tried to find more specific and simpler phrasing. However, using verbs in my variant is hanging in the air.